config/filter.d/nginx-http-auth.conf

```diff
@@ -3,15 +3,23 @@
 
 [Definition]
 
+extendregex = 
 
 failregex = ^ \[error\] \d+#\d+: \*\d+ user "\S+":? (password mismatch|was not found in ".*"), client: <HOST>, server: \S*, request: "\S+ \S+ HTTP/\d+\.\d+", host: "\S+"(, referrer: "\S+")?\s*$
+            %(extendregex)s
 
 ignoreregex = 
 
+protect_web_apps = ^ \[error\] \d+#\d+: \*\d+ user \S* ?(not found|not authorized|password mismatch|sent invalid token|misbehaved): .+, client: <HOST>
```

Comment:
> could HOST be anchored at the end here due to .+ preceding it?

No, still worse (because configurable in nginx-config and per default contains referer etc. so user input).
Imho, it is better to resolve greedy `.+` with something plausible